benchmarks/benchmark-http2.js

```diff
@@ -15,7 +15,7 @@ const servername = 'agent1'
 const iterations = (parseInt(process.env.SAMPLES, 10) || 10) + 1
 const errorThreshold = parseInt(process.env.ERROR_THRESHOLD, 10) || 3
 const connections = parseInt(process.env.CONNECTIONS, 10) || 50
-const pipelining = parseInt(process.env.PIPELINING, 10) || 10
+const maxConcurrentStreams = parseInt(process.env.MULTIPLEXING, 10) || 10
```

Comment:
Either this or:
```suggestion
const maxConcurrentStreams = parseInt(process.env.MAX_CONCURRENT_STREAMS, 10) || 10
```

Unless you would prefer re-using the same semantic across benchmarks and keep "pipelining" here.